frontend/src/utils/i18n.ts

```diff
@@ -13,17 +13,17 @@ const LOCALES = {
     flag: "fr",
     name: "Français",
     locale: "fr"
-  }
-  /* 'de': {
-    flag: 'de',
-    name: 'Deutsch',
-    locale: 'de',
   },
-  'ja': {
-    flag: 'jp',
-    name: '日本語',
-    locale: 'ja',
-  }, */
+  de: {
+    flag: "de",
+    name: "Deutsch",
+    locale: "de"
+  },
+  ja: {
+    flag: "jp",
+    name: "日本語",
+    locale: "ja"
+  }
```

Comment:
This souldn't be added yet

Reply:
> This souldn't be added yet

I added english placeholders for every missing language. If we are talking about actually finished translations that would only be french and english

Reply:
fixed